### DISCUSSION TITLE
Noob here!

### DISCUSSION THREAD

**Discussion by calmination** [dsharlet/LiveSPICE#169] [Ideas]
Hi! i am getting into audio circuit stuff, and i am trying to learn how to do some basic audio circuits.
I was wondering if it's possible to add some components needed for some basic oscillator building like for example 40106 hex schmitt trigger inverters.

**Comment by dsharlet** [comment]
The two ways of getting something like this into a simulation is either developing a model and a new component type that implements the model, or building a proxy circuit. For example, there is the "Op-Amp Model.schx" circuit that is included as an example and shows how op-amps could be modeled.

This particular component looks like it would be quite challenging to simulate efficiently enough for real time without some simplifications. I only skimmed the datasheet so I could be wrong.

BTW, there is another example "Wien Bridge Oscillator.schx" that might be interesting to you (if you just want some way of implementing an oscillator).
